src/arguments.cpp

```diff
@@ -163,6 +163,9 @@ Error Arguments::parse(const char* args) {
             CASE("version")
                 _action = ACTION_VERSION;
 
+            CASE("metrics")
+                _action = ACTION_METRICS;
```

Comment:
Do we want to keep `meminfo`? `metrics` provide the same (and more) information in human readable format